code/src/Components/Form.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+
+
+
+const Form = () => {
+  const [genre, setGenre] = useState('');
+  const [time, setTime] = useState('');
+  const [experience, setExperience] = useState('');
+  const [surveyComplete, setSurveyComplete] = useState(false);
+
+  const timeChoice = ['Full time', 'Part time', 'Temporary project']
+  const experienceChoice = ['Under education', 'Junior', 'Senior']
+
+  const isSurveyComplete = () => {
+    if(time !== '' && genre !== '' && experience !== '' ){
+        return true;
+     }
+    return false; 
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault() 
+    
+    if (isSurveyComplete()) {
+      setSurveyComplete(true)
+    } else {
+      alert("Please choose options for each question")
+    }
```

Comment:
Nice that you give the reason to a user why its not submitting 👌 , always frustrating when not working.